styles.css

```diff
@@ -47,11 +47,33 @@ body {
   color: #172b4d;
   font-family: "Poppins", sans-serif;
   font-style: normal;
-  font-size: 55px;
+  font-size: 50px;
   line-height: 52px;
   margin-bottom: 20px;
 }
 
+.work p {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin: 16px;
+}
+
+.section4 p {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  text-align: center;
+  font-weight: 400;
+  width: 327px;
+  font-size: 20px;
+  line-height: 28px;
+  color: #fff;
+  margin-top: 24px;
+}

```

Comment:
Please consider using the same color as used in the Figma design to make it look alike to the design.


### Color code:-

```
.section4 p {
  color: #EBEBFF;
}
```